docs/src/index.rst

```diff
@@ -1,18 +1,22 @@
 iris-esmf-regrid
 ================
 
-A collection of structured and unstructured ESMF_ regridding schemes for Iris_.
+Iris-esmf-regrid is a package which aims to bridge the gap between Iris_ data
+handling and ESMF_ regridding. This is done largely by way of providing
+Iris_-like regridding schemes.
 
-This rendered documentation is so far limited to iris-esmf-regrid's API. Some
-basic examples and change logs can be found on GitHub_.
+This documentation is a work in progress and some pages will be incomplete.
 
 :doc:`Click to view the API<_api_generated/modules>`
 
 .. toctree::
    :hidden:
 
+   installing
+   userguide/index
+   developers_guide/index
    _api_generated/modules
+   whatsnew/index
```

Comment:
This new documentation is our first opportunity to try the structure laid out in the [Divio documentation system](https://documentation.divio.com/).

I like the heading titles in the [Matplotlib docs](https://matplotlib.org/stable/index.html), which are one example of implementing the Divio system:

![image](https://github.com/SciTools-incubator/iris-esmf-regrid/assets/40734014/045dd87a-e84e-4be3-b447-fbd76f88da00)

Reply:
As far as the overall structure goes, I believe that the Divio system would make most sense as something to apply structure *along side* the existing structure of the heading bar. For example, with the API existing as its own clearly labeled thing in the heading bar but also listed as reference documentation with respect to the Divio system. I've rewritten the index page to make the more clear.